doc/DOCUMENTATION.org

```diff
@@ -1196,6 +1196,13 @@ pdf-view-mode, text-mode as well as buffers over 1Mb:
                                            :size-limit-kb 1000))
 #+END_SRC
 
+Enable line numbers everywhere, except for buffers over 1Mb:
+
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-lines-numbers '(:relative nil
+                                             :size-limit-kb 1000))
+#+END_SRC
+
```

Comment:
Good example, I would maybe put it just after the next one (to keep enabled and disabled examples together).